aten/src/ATen/LegacyTHFunctionsCPU.cpp

```diff
@@ -165,72 +165,6 @@ Tensor _th_nonzero(const Tensor & self) {
     }
     return result;
 }
-Tensor & _th_index_copy_(Tensor & self, int64_t dim, const Tensor & index, const Tensor & source) {
-    // DeviceGuard omitted
-    auto dispatch_scalar_type = infer_scalar_type(self);
```

Comment:
check out `git grep indexCopy` and remove these entries as well.

Reply:
Ooops, sorry, did not update the page to see the rest having been already removed.